### ISSUE TITLE
[Test] Add tests for 'conv2d' with 'dilations' option

### ISSUE THREAD

**Issue by Honry** [webmachinelearning/webnn-polyfill#84]
While developing DeepLab V3 MobileNet V2 example, I captured a bug when using 'conv2d' with 'dilations' option and identified it is a `tfjs-webgl-backend` issue, see details at https://github.com/tensorflow/tfjs/issues/5293.

So we may need to add relevant tests to cover the gap, besides above checkpoint, tests from https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/ops/depthwise_conv2d_test.ts maybe good examples for making good test coverage.

### Where I capture the issue?

I covert the frozen graph from http://download.tensorflow.org/models/deeplabv3_mnv2_pascal_trainval_2018_01_29.tar.gz to a tflite model, and found the issue at node of `MobilenetV2/expanded_conv_14/depthwise/depthwise`.

### Code snippet to reproduce
```
    const input = tf.fill([1, 65, 65, 960], 0, 'float32');
    const weights = tf.fill([3, 3, 960, 1], 1, 'float32');
    const result = tf.depthwiseConv2d(input, weights, 1, 'same', 'NHWC', 4);
```

**Comment by BruceDai** [comment]
@Honry Thanks for report this issue,  I will add some 'conv2d' tests with 'dilations'.

**Comment by BruceDai** [comment]
Fixed by #88, close it.
